app/mm-redux/selectors/entities/roles.ts

```diff
@@ -154,7 +156,7 @@ export const getMyChannelPermissions = reselect.createSelector(
                     for (const permission of roles[roleName].permissions) {
                         permissions.add(permission);
                     }
-                    roleFound = true && teamRoleFound;
+                    roleFound = true && (teamRoleFound || teamId === undefined);
                 }
```

Comment:
These changes were unrelated but fix an issue where if you call `haveIChannelPermission` but dont pass a team id and set the default to true then it always returns true (since the empty team id never returns a team role resulting in roleFound = false)